oshi-demo/src/main/java/oshi/demo/DetectVM.java

```diff
@@ -47,11 +49,12 @@
         vmMacAddressOUI.put("00:0F:4B", "Virtual Iron 4");
         vmMacAddressOUI.put("00:16:3E", "Xen or Oracle VM");
         vmMacAddressOUI.put("08:00:27", "VirtualBox");
+        vmMacAddressOUI.put("02:42", "Docker Container");
```

Comment:
An OUI is 24 bits, so this should be `02:42:AC`.  Your source only uses the first two bytes because the remaining 4 bytes are drawn from the IP address; however, the Docker IP range is `172.17.0.0/16` meaning the MAC always starts `02:42:AC:11` and only the last 16 bits vary.

Reply:
@dbwiddis Thanks for your feedback.
I push some modification.